bindings/python/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ set(${PYWRAP}_SOURCES
     contacts/contact-6d.cpp
     contacts/contact-point.cpp
     contacts/contact-two-frame-positions.cpp
+    contacts/measured-6Dwrench
```

Comment:
missing file extension
I would rename the file following the naming convention of the project.